delivery/models/project.py

```diff
@@ -3,15 +3,31 @@
 from delivery.models import BaseModel
 
 
-class Project(BaseModel):
+class BaseProject(BaseModel):
+    """
+    Base class for the different project models
+    """
+
+    def __eq__(self, other):
+        """
+        Two project should be considered the same if the represent the same directory on disk
+        :param other: instance of RunfolderProject
+        :return: true if the same project, otherwise false
+        """
+        if isinstance(other, self.__class__):
+            return self.path == other.path
```

Comment:
Also a good point, I think everywhere the paths are instantiated it's a absolute path, but I'll double check.